app/src/main/java/mattecarra/accapp/activities/AccConfigEditorActivity.kt

```diff
@@ -59,6 +59,7 @@ class AccConfigEditorActivity : ScopedAppActivity(), NumberPicker.OnValueChangeL
         setContentView(binding.root)
 
         content = binding.contentAccConfigEditor
+        content = binding.content
```

Comment:
wrong add.. sorry. Need `content = binding.contentAccConfigEditor` For:
`<include layout="@layout/content_acc_config_editor"  android:id="@+id/content_acc_config_editor"/>`